changelogs/fragments/20240707-postgresql_admin_manage_password.yml

```diff
@@ -0,0 +1,2 @@
+bugfixes:
+  - Adding admin password requirements of azure_manage_postgresql_postgresql_admin_password role to README file
```

Comment:
```suggestion
  - Adding admin password requirements of azure_manage_postgresql role to README file
```

Reply:
or
```suggestion
  - Adding admin password requirements of azure_manage_postgresql_postgresql_admin_password var to README file of azure_manage_postgresql role
```

Reply:
applied this change:  Adding admin password requirements of azure_manage_postgresql role to README file